pkg/skaffold/schema/latest/config.go

```diff
@@ -118,6 +119,23 @@ type KanikoBuild struct {
 	Image          string             `yaml:"image,omitempty"`
 }
 
+// AzureContainerBuild contains the fields needed to do a build
+// on Azure Container Registry
+type AzureContainerBuild struct {
+	Credentials       AzureContainerBuildCredentials `yaml:"credentials,omitempty"`
+	ContainerRegistry string                         `yaml:"containerRegistry,omitempty"`
```

Comment:
Could a few of those field have sensible default values or be, by default, derived from the image name. Like if I push an image called `myregistry.azurecr.io/skaffold-example` would it make sense to set `containerRegistry` to `myregistry` by default?

That's something we do on GCB to make sure the Skaffold config is as short as possible and contains as few duplication as required.

Reply:
Yes, I can get the container registry name from the image name, but the resource group needs to be provided separately since it may differ from the image name.